template/pyproject.toml.jinja

```diff
@@ -13,7 +13,75 @@ authors = [
 ]
 classifiers = [
   "Development Status :: 4 - Beta",
+[%- if copyright_license == 'Academic Free License v3.0' %]
```

Comment:
Let us just align the values with PyPI classifiers so that we can use it directly here rather than so many if-else logic.